rpc/ws/client.go

```diff
@@ -172,17 +172,37 @@ func (c *Client) handleMessage(message []byte) {
 	// when receiving message with id. the result will be a subscription number.
 	// that number will be associated to all future message destine to this request
 
+	// Check for an error in the message.
+	if errorCode, errMsg, ok := getJsonRpcError(message); ok {
```

Comment:
https://github.com/gagliardetto/solana-go/blob/main/rpc/jsonrpc/jsonrpc.go#L198

Wouldnt it be possible to reuse RPCResponse here?